app/app_main.c

```diff
@@ -354,6 +355,11 @@ static int ingest_cli(APP_CONFIG *cfg, int argc, char **argv) {
     argv++;
     argc--;
     while (argc >= 1) {
+        /* version option used by itself, ignore remaining command line */
+        if (strncmp(*argv, "--version", strlen("--version")) == 0) {
+            printf("\nACVP version %s(%s)\n", acvp_version(), acvp_protocol_version());
```

Comment:
From the cli ouput, a user might not know which number is the library vs protocol?

Reply:
Something like this better ?

ACVP library version(protocol version): 1.0.0(0.4)